lint/private/patcher.mjs

```diff
@@ -0,0 +1,55 @@
+import childProcess from 'child_process';
+import path from 'path';
+import fs from 'fs';
+import os from 'os';
+
+const node = process.argv[0];
+const wrapper = process.argv[1];
+const linterArgs = process.argv.slice(2);
+
+const filesToLint = fs.readFileSync(process.env.FILES_TO_LINT_PATH).toString().replace(/\n+$/, '').split('\n');
+
+const tmpDirPath = fs.mkdtempSync("/tmp/");
+
+const tmpFiles = [];
+for (const f of filesToLint) {
+    const destF = path.join(tmpDirPath, f);
+    fs.mkdirSync(path.dirname(destF), {recursive: true});
+    fs.cpSync(f, destF);
+    fs.chmodSync(destF, "600");
+    tmpFiles.push(destF);
+}
+
+const filesIndex = linterArgs.indexOf("$$FILES");
+linterArgs.splice(filesIndex, 1, ...tmpFiles);
+
+const linterPath = path.join(process.env["JS_BINARY__EXECROOT"], process.env["LINTER_PATH"]);
+const newEnv = {...process.env};
+delete newEnv["JS_BINARY__EXIT_CODE_OUTPUT_FILE"];
```

Comment:
I think you won't need this anymore after rebase

Reply:
Addressed in the updated implementation.